app/jobs/scheduled_mailings_delivery_job.rb

```diff
@@ -1,7 +1,17 @@
 class ScheduledMailingsDeliveryJob < ApplicationJob
   queue_as :default
 
+  DEFAULT_DELAY = 2.minutes.freeze
```

Comment:
Can't this be moved to a constants or config file for reuse both here and in `app/jobs/email_monitor_job.rb` ?

Reply:
I think we want them separate in case we ever want different delays on the two jobs.

Reply:
cool, thought that might be the case